src/main/java/org/jabref/gui/linkedfile/LinkedFileDialogController.java

```diff
@@ -18,7 +18,10 @@
 import com.airhacks.afterburner.views.ViewLoader;
 import jakarta.inject.Inject;
 
-public class LinkedFileEditDialogView extends BaseDialog<LinkedFile> {
+public class LinkedFileDialogController extends BaseDialog<LinkedFile> {
+
+    private static final ButtonType ADD_BUTTON = new ButtonType(Localization.lang("Add"), ButtonType.OK.getButtonData());
+    private static final ButtonType EDIT_BUTTON = ButtonType.APPLY;
```

Comment:
Go through the flow once. These two buttons have no difference in functionality since in the if condition in line 72, the same action is triggered (apply) regardless of which of these buttons are chosen. No need of two different button types.

Reply:
I added two different buttons because in the previous(closed) PR, the requirements mentioned "Add" button for add a file dialog and "Apply" button for edit dialog. Though both have same functionality, I made them separate in order to meet that requirement
![Screenshot 2024-07-31 221235](https://github.com/user-attachments/assets/c37cf0a0-3ce0-4a2f-8a60-4e1b264990d6)
I can change it if it doesn't make any difference, Please let me know. 

Thanks